core/src/main/scala/com.mediative.amadou/Stage.scala

```diff
@@ -89,6 +89,11 @@ object Stage {
     }
   }
 
+  def trySequence[S, T](stages: Seq[Stage[S, T]]): Stage[S, Seq[Try[T]]] = new Stage[S, Seq[Try[T]]] {
+    override def name = "trySequence"
+    override def run(ctx: Stage.Context[S]) = Try(stages.map(_.run(ctx)))
```

Comment:
Actually here, I need all or nothing, so it should try to run all, and fail with some custom error message if there is one failure.